xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererDXVA.cpp

```diff
@@ -144,9 +144,9 @@ bool CRendererDXVA::Configure(const VideoPicture& picture, float fps, unsigned o
           m_enumerator->SupportedConversions(m_conversionsArgs);
       if (!conversions.empty())
       {
-        const ProcessorConversion chosenConversion = ChooseConversion(
-            conversions, picture.colorBits,
-            static_cast<AVColorTransferCharacteristic>(picture.color_transfer), m_tryVSR);
+        const ProcessorConversion chosenConversion =
+            ChooseConversion(conversions, picture.colorBits,
+                             static_cast<AVColorTransferCharacteristic>(picture.color_transfer));
```

Comment:
The member variable was created with the idea of reacting to settings changes during playback with CheckVideoParameters(), but I stopped half-way in the PR, and the value is calculated only once in Configure. I should have removed the member, which is what I suggest you instead.

Reply:
The commit is fine. The member is already used in CheckVideoParameters, the editor didn't highlight it earlier and I missed it.